gui/MetadataView.cxx

```diff
@@ -296,6 +299,11 @@ void MetadataView::updateMetadata(kv::metadata_vector const& mdVec)
 
       if (mdi)
       {
+        if (!keys.contains(k))
+        {
+          auto const tag = static_cast<kv::vital_metadata_tag>(k);
+          d->addItem(k, qtString(traits.tag_to_name(tag)));
+        }
         d->setItemValue(k, qtString(mdi->as_string()));
```

Comment:
If `addItem` is called, is this not redundant? (Should it be in an `else`?)

Reply:
As far as I can tell, `addItem` only creates an entry in the map, but doesn't set the value and `setItemValue` only sets the value if the item already exists.  So I always want to set the item value, but I only want to add the item if it's not already in the map.

Reply:
Ah, right. I just saw that text was being passed to both. I didn't look closely at *what* text, or go remind myself what those methods actually do. You're right, though; the first takes the *key* text, the second takes the *value* text, so yeah, obviously calling both is correct.